dockerfiles/evmapp/entrypoint.sh

```diff
@@ -29,30 +29,30 @@ export SCNODE_REMOTE_KEY_MANAGER_ENABLED
 
 
 # Function(s)
+fn_die() {
+  echo -e "$1" >&2
```

Comment:
Want to use our new pattern better ?
```
echo -e "\n\033[1;31m${1}\033[0m\n" >&2
```

https://github.com/HorizenOfficial/compose-evm-simplified/blob/7c0d262a6b8136a693b42e188b68c44efdaddc53/scripts/utils.sh#L3-L6